src/jitter.rs

```diff
@@ -159,32 +190,17 @@ impl<T: JitterTimer+Clone> JitterRng<T> {
     ///
     /// This method is more low-level than `new()`. It is the responsibility of
     /// the caller to run `test_timer` before using any numbers generated with
-    /// `JitterRng`, and optionally call `set_rounds()`.
+    /// `JitterRng`, and optionally call `set_rounds()`. Also it is important to
+    /// call `gen_entropy` once before using the first result to initialize the
+    /// entropy collection pool.
```

Comment:
This comment isn't true because `gen_entropy` is called within this function already!